python/python/lancedb/query.py

```diff
@@ -206,8 +207,6 @@ def _resolve_query(cls, table, query, query_type, vector_column_name):
         elif query_type == "auto":
             if isinstance(query, (list, np.ndarray)):
                 return query, "vector"
-            if isinstance(query, tuple):
-                return query, "hybrid"
```

Comment:
can we remain this? or it's a breaking change

Reply:
this will be a breaking change. The tuple pathway was temporary for internal benchmarking while we didn't have the API finalised